src/NuGet.Clients/NuGet.PackageManagement.UI/PackageInstallationInfo.cs

```diff
@@ -119,6 +119,11 @@ public override int GetHashCode()
             return StringComparer.OrdinalIgnoreCase.GetHashCode(_projectName);
         }
 
+        public override string ToString()
```

Comment:
Yes this is what Narrator reads and what shows up in Inspect.  When the version is null, it would just show "WpfApp1" instead of "WpfApp1 10.0.2.0"
I will change it to use the not installed resource string then